### DISCUSSION TITLE
is there a way to disable dark mode?

### DISCUSSION THREAD

**Discussion by jaxramus** [laravel/nova-issues#4371] [Q&A]
is there a way to disable dark mode?

**Comment by crynobone** [comment]


**Reply by nrueckmann** [reply]
This feature was added in [v4.4.0](https://nova.laravel.com/releases/4.4.0)

**Reply by jaxramus** [reply]
> v4.4.0

That feature merely disables the dropdown and uses the system default. @crynobone solution enforces light mode regardless of system setting, which is what i wanted.

**Comment by jerothor** [comment]
just fyi this answer also works for the login page (crynobones unfortunately does not apply to the login page):

https://stackoverflow.com/a/75156840/21064320
